Sources/ApolloCodegenLib/Frontend/JavaScript/src/utilities/apolloCodegenSchemaExtension.ts

```diff
@@ -2,10 +2,10 @@ import { DirectiveDefinitionNode, DocumentNode, Kind, NameNode, StringValueNode
 
 const directive_apollo_client_ios_localCacheMutation: DirectiveDefinitionNode = {
   kind: Kind.DIRECTIVE_DEFINITION,
-  description: stringNode("A directive used by the Apollo iOS client to annotate queries that should be used for local cache mutations instead of standard query operations."),
+  description: stringNode("A directive used by the Apollo iOS client to annotate operations or fragments that should be used exclusively for generating local cache mutations instead of as standard operations."),
```

Comment:
Do you think we might need to clarify further that these will be separate data structures to regular operations? "used exclusively for" is quite telling already though. 🤔

Reply:
Yeah, that's why I added "exclusively". I think we should make that clear in documentation. Very few people are ever going to be looking at the directive description here.